src/main/java/com/rocketchat/core/APIServer.java

```diff
@@ -0,0 +1,22 @@
+package com.rocketchat.core;
+
+import com.rocketchat.models.chat.Chat;
+import com.rocketchat.models.user.User;
+import com.rocketchat.storage.InMemoryStorage;
+import com.rocketchat.storage.Storage;
+import com.rocketchat.websocket.core.BigQueue;
+import com.rocketchat.websocket.producers.MessageProducer;
+import com.rocketchat.websocket.producers.Producer;
+
+public class APIServer implements ServerBuilder {
```

Comment:
Querés que tu clase sea extensible o que puedas heredarla? Si no querés, hacela `final` así:

```suggestion
public final class APIServer implements ServerBuilder {
```